TestVectorsAwsCryptographicMaterialProviders/Makefile

```diff
@@ -254,3 +254,31 @@ test_decrypt_encrypt_vectors_rust:
 
 test_decrypt_encrypt_vectors_go:
 	go -C runtimes/go/ImplementationFromDafny-go run ImplementationFromDafny.go decrypt --manifest-path=..
+
+# Fuzz Testing Commands
+
+# Generate fuzzed test vectors in the Python runtime directory (50 vectors)
+test_generate_fuzz_vectors:
+    cp dafny/TestVectorsAwsCryptographicMaterialProviders/test/keys.json runtimes/python/
+    cd runtimes/python && python3 fuzz_generator.py --num-vectors 50
+    @echo "fuzzed test vectors generated in runtimes/python/"
```

Comment:
This echo will only add noise. When this command succeeds its clear it has generated

Reply:
This is blocking too. We either need to remove this if it does not add any value. If it adds value then add it to all the existing make file commands.

Reply:
Yeah sure. If you have a TODO its fine but don't let all the TODO drown you. You goal is to have this running in main not in any other branch